### PR TITLE
OPS-0 Changes for dynamodb conditionals and rds option/parameter groups

### PR DESCRIPTION
**Pull request by runderwoodcr14**
The following 
- dynamodb_enabled
-dynamodb_enable_autoscaler

Are already boolean variables, it doesn't need the conditional expression, this variables are initialised as false by default.

There is a [commit](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/bdc2de20ced43490d02f61be4c704eae282785fb) in the module for rds that has not been yet merged into master this will avoid the problem of the creation by default of the option groups and parameter groups.
To avoid this the variables:
- rds_option_group_name
- rds_parameter_group_name

Were set to the value `default` instead of an empty string

### CODE REVIEW

**Review by ronny-panknin-flaconi — DISMISSED** [review]
wha nice 👍

**Review by cytopia — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -646,7 +646,7 @@ variable "rds_iam_database_authentication_enabled" {
 variable "rds_parameter_group_name" {
   description = "Parameter group for database"
   type        = string
-  default     = ""
+  default     = "default"
```
Comment (danvaida) [inline variables.tf:649]: We could've also tested for an empty string.

**Review by danvaida — DISMISSED** [review]

File: variables.tf (hunk 1 above)
Comment (cytopia) [reply variables.tf:649]: I think "default" nails it, as the default group is named by this as well

Comment (danvaida) [reply variables.tf:649]: @cytopia thought about this, too. I don't think it's a good idea to try and mimic the resource names that AWS is handing out themselves. It's always good to have a good distinction between the two, even though the config is identical to the **real** default. Luckily we can look at the tags to see if it's TF that's handling that.

I'll give you an example: the special case of the default VPC.

Comment (runderwoodcr14) [reply variables.tf]: As always this can be improved, or now it was to solve a problem that in the future will be addressed by the module itself once they merge the changes(if they ever do it)

**Review by cytopia — APPROVED** [review]
